frontend/src/components/Form/index.jsx

```diff
@@ -1,7 +1,14 @@
+import React, { useState } from "react";
 import Title from "@components/Archi/Title";
+import Modal from "@components/Modal";
 import SForm from "./style";
 
 export default function Form() {
+  const [message, setMessage] = useState("");
+  const [name, setName] = useState("");
+  const [object, setObject] = useState("");
+  const [email, setEmail] = useState("");
```

Comment:
We saw how to create one state to store all field's information, to make the next steps easier